xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -425,11 +433,11 @@ bool CGUIWindowVideoBase::ShowInfo(const CFileItemPtr& item2, const ScraperPtr&
     *item->GetVideoInfoTag() = movieDetails;
     pDlgInfo->SetMovie(item.get());
     pDlgInfo->Open();
-    if (pDlgInfo->HasUpdatedUserrating())
-      return true;
+
     needsRefresh = pDlgInfo->NeedRefresh();
     if (!needsRefresh)
-      return pDlgInfo->HasUpdatedThumb();
+      return pDlgInfo->HasUpdatedThumb() || pDlgInfo->HasUpdatedUserrating();
+
```

Comment:
That change is not exactly equivalent (hasupdateduserrating = true with needsrefresh = false) but not needed here when handling in caller anyway.

However the part at the bottom of the loop has to move into the caller as well.